department-of-veterans-affairs/caseflow
# Add search by Veteran ID to /jobs page

Resolves #13359


### Description
Add a search field on the "Background Jobs" admin page to filter results by veteran file number.

### Acceptance Criteria
- [ ] Search field added to the rendered page
- [ ] Async jobs controller handles the query by returning only the matching job(s)

### Testing Plan
1. Navigate to `/jobs`
2. In the search bar, input a veteran's file number
3. Verify that the results displayed on the job page are only for the veteran file number search input

### User Facing Changes

 BEFORE|AFTER
 ---|---
<img width="1271" alt="Screen Shot 2020-09-28 at 7 49 38 PM" src="https://user-images.githubusercontent.com/68879427/94507342-62ef4d80-01c4-11eb-89f1-c06774bf9e84.png">
|
<img width="986" alt="Screen Shot 2020-10-05 at 4 39 11 PM" src="https://user-images.githubusercontent.com/68879427/95154983-f0441c00-0747-11eb-892a-b92e0ad5fa63.png">
